commerce-account-admin-web/src/main/java/com/liferay/commerce/account/admin/web/internal/display/context/CommerceAccountUserRelAdminDisplayContext.java

```diff
@@ -140,6 +143,55 @@ public PortletURL getPortletURL() {
 		return _searchContainer;
 	}
 
+	public String getUserRoleIds(CommerceAccountUserRel commerceAccountUserRel)
+		throws PortalException {
+
+		List<UserGroupRole> userGroupRoles =
+			commerceAccountUserRel.getUserGroupRoles();
+
+		Stream<UserGroupRole> stream = userGroupRoles.stream();
+
+		Long[] rolesArray = stream.map(
+			UserGroupRole::getRoleId
+		).toArray(
+			Long[]::new
+		);
+
+		return StringUtil.merge(rolesArray, StringPool.COMMA_AND_SPACE);
```

Comment:
This could all be replaced with `return ListUtil.toString(commerceAccountUserRel.getUserGroupRoles(), "roleId", StringPool.COMMA_AND_SPACE)` :)

Reply:
Perfect, didn't realize this method was an option. There are probably a few places we can replace streaming with this.